### ISSUE TITLE
Sync the animation time with the loading bar

### ISSUE THREAD

**Issue by evromalarkey** [KoffeinFlummi/AGM#2088]
**Version**: 0.95.2

Most of the animations with different actions as bandaging are not synced to the loading bar / waiting time.
Perhaps would be better to cycle the animation, until action is complete?

*eg. You are bandaging someone and you are not medic. The animation will stop in the middle of bandaging.*

**Comment by BryanBoru** [comment]
I agree, This would decrease the likelihood of people believing they are
done and therefore accidently cancelling the mending. which is what ive
been seeing
